src/cljs/main/broadfcui/components/modals.cljs

```diff
@@ -0,0 +1,66 @@
+(ns broadfcui.components.modals
+  (:require
+   [broadfcui.common :as common]
+   [broadfcui.common.components :as comps]
+   [broadfcui.common.style :as style]
+   [dmohs.react :as r]
+   [org.broadinstitute.uicomps.modal :as modal]
+   ))
+
+(r/defc OKCancelForm
+  {:get-default-props
+   (fn []
+     {:show-cancel? true
+      :show-close? true})
+   :render
+   (fn [{:keys [this props]}]
+     (let [{:keys [header content dismiss ok-button show-cancel? cancel-text show-close?]} props
+           cancel-text (or cancel-text "Cancel")]
+       (modal/render
+        {:content
+         [:div {}
```

Comment:
Can we deal with all of the weird percentages and arbitrary non-rem pixel values while we're in here?